nix/default.nix

```diff
@@ -144,6 +144,8 @@ in
         else "RelWithDebInfo";
 
       cmakeFlags = [
+        # precompiled headers are useless in Nix
+        "-DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON"
```

Comment:
Perhaps we can use cmakeBool here

Reply:
I did not think it would work, but it does. Added.